Services/Seed.cs

```diff
@@ -162,7 +162,14 @@ public class Seed : ISeed
             {
                 try
                 {
-                    await this.telemetryClient.UpdateRuleAsync(rule, "*");
```

Comment:
I believe the telemetry service was intended to have an idempotent PUT request for a rule with a specified ID that would create a new rule if it didn't exist. Did the behavior change in the Telemetry service?

Reply:
idempotent PUT means that it should not create a new rule if doesn't exist (because in first call it will create and in the very next call it will update the result, hence it won't be idempotent anymore). The telemetry service relies on Storage adapter which is actually creating a key value pair if it doesn't exists on PUT.